tests/integration_tutorials/c/export_info.c

```diff
@@ -30,8 +33,10 @@ int main()
 
     // Connecting using the connection settings
     CoSimIO_Info connect_info = CoSimIO_Connect(connection_settings);
+    char connection_name[BUFSIZ];
+    strcpy(connection_name, CoSimIO_Info_GetString(connect_info, "connection_name"));
```

Comment:
In cases like this, why we should copy the connection_name? What was the problem of directly using the pointer to the internal name stored in the info?

Reply:
I thought it is easier to handle when we have a local variable, then we can safely delete the `connect_info`

Otherwise we need to keep it around until we disconnect

Not sure what is better, we can also leave it as is and let the user decide what they prefer

Reply:
I see... I think we should use the strcpy only when we want to keep some string for our own. In cases like the above one, I would just use the pointer to the internal data and put a comment in the tutorial that `// p_connection_name is not valid anymore ` after freeing the info

Reply:
ok